lib/engines/content_block_manager/app/helpers/workflow_helper.rb

```diff
@@ -0,0 +1,15 @@
+module WorkflowHelper
+  def back_path(content_block_edition, current_step)
+    if current_step == "review" && content_block_edition.document.is_new_block?
+      content_block_manager.content_block_manager_content_block_documents_path
```

Comment:
now I'm questing why we take users back to the documents page when reviewing a new block - it seems weird, when my browser button just takes me back to the editing form. Is it because we can't actually provide a link back to the form including the data they may have changed?

Reply:
Yeah, I thought that was weird too. I guess we could throw them back to the edit state for that block. We'd just need to be careful that we don't show them the rest of the edit flow. I think it's doable though. Lemme have a look.

Reply:
Hmmm. I think I'll merge as is. I'll make a card to pick it up at a later date.